deployment/config.go

```diff
@@ -53,6 +59,9 @@ func (c *Config) IsValid() error {
 	if len(clusterName) == 0 || !unicode.IsLetter(firstRune) || !isAlphanumeric(clusterName) {
 		return fmt.Errorf("db cluster name must begin with a letter and contain only alphanumeric characters")
 	}
+	if c.AgentCount < 2 {
+		return fmt.Errorf("number of agents must be greater than 2")
+	}
```

Comment:
Is there a technical reason for this? Everything can work fine with 1 agent as far as I know.

Reply:
This one remained from the initial approach we can change this to `1` 👍